build/package/Installer.DEB.targets

```diff
@@ -42,9 +42,11 @@
       <ManpagesDirectory>$(RepoRoot)/Documentation/manpages</ManpagesDirectory>
       <EndToEndTestProject>$(RepoRoot)/test/EndToEnd/EndToEnd.csproj</EndToEndTestProject>
       <CLISdkRoot>$(OutputDirectory)/sdk</CLISdkRoot>
+      <AspNetRuntimeRoot>$(AspNetRuntimePackageStorePublishDirectory)</AspNetRuntimeRoot>
```

Comment:
I think it is a good pattern of the file that it list out all the 4 different pieces that will get into to the deb package process in this PropertyGroup. And the next PropertyGroup select all the files

But it is xml... I have no strong opinion